classes/class-liveblog.php

```diff
@@ -285,6 +288,31 @@ public static function jetpack_search_es_query_args( $args, $query ) {
 			return $args;
 		}
 
+		/**
+		 * Remove <script> tag from Twitter embeds.
+		 *
+		 * @param string $html           The original HTML returned from the external oembed/embed provider.
+		 * @param string $url            The URL found in the content.
+		 * @param mixed  $attr           An array with extra attributes.
+		 * @param int    $post_id        The post ID.
+		 * @return string The filtered HTML.
+		 */
+		public static function embed_oembed_html( $html, $url, $attr, $post_id ) {
+			if ( 0 !== strpos( $url, 'https://twitter.com' ) ) {
+				return $html;
+			}
+
+			$rest_route  = $GLOBALS['wp']->query_vars['rest_route'];
```

Comment:
Is there a chance that `query_vars['rest_route']` would throw an index error if not in a `REST_REQUEST`?